openthread/cmake/extensions.cmake

```diff
@@ -3,20 +3,44 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
+include(${NRFXLIB_DIR}/common.cmake)
+include(${NRFXLIB_DIR}/nrf_security/cmake/extensions.cmake)
```

Comment:
we should not include extensions from extensions.

please move all `include(<path>/extensions.cmake)` to toplevel CMakeLists.txt:
https://github.com/nrfconnect/sdk-nrfxlib/blob/master/CMakeLists.txt

so that we have a single location.
This will also align with how extensions are loaded in `sdk-nrf`

Reply:
Done for common.cmake. 
openthread/cmake/extension.cmake is only for internal use. I don't think it should be loaded if openthread is not enabled.